css/style.css

```diff
@@ -0,0 +1,21 @@
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
+    padding: 0;
```

Comment:
You could add margin and padding to the universal selector instead if you like to. Just a suggestion. Than you can start from scratch with it as well.

Reply:
Changed - Thanks! :)